crates/bevy_app/src/app.rs

```diff
@@ -183,6 +183,8 @@ impl Default for App {
         app.init_resource::<AppTypeRegistry>();
 
         app.add_plugins(MainSchedulePlugin);
+        app.add_systems(FixedUpdate, bevy_ecs::event::event_queue_update_system);
```

Comment:
adding a system in app feels off. I don't think we're adding any other systems here. Would this make more sense in bevy_time?

Reply:
Yeah, it would. I'll move it.